src/store/actions.ts

```diff
@@ -7,6 +7,7 @@ import getPricingPolicies from "@/utils/getPricingPolicies";
 import createDataRequests from "@/utils/createDataRequests";
 import isNodeOnline from "@/utils/isNodeOnline";
 import getChainData from "@/utils/getChainData";
+import { toArray } from "lodash";
```

Comment:
what is this for ?

Reply:
I think it was automatically imported, I have removed it and other unused imports